__tests__/users.test.js

```diff
@@ -0,0 +1,44 @@
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const app = require("../app");
+const {
+  topicData,
+  userData,
+  articleData,
+  commentData,
+} = require("../db/data/test-data/index");
+
+beforeEach(() => {
+  return seed({ topicData, userData, articleData, commentData });
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("/api/users", () => {
+  test("200 - GET - will return an article object with the given article_id, with the properties of author, title, article_id, body, topic, created_at, votes and article_img_url", () => {
```

Comment:
Is this what you're testing for? Sounds like you want users not an article object. Don't copy and paste tests, you need to be able to know how to write them.